R/groupComparisonQCPlots.R

```diff
@@ -49,20 +49,35 @@
 #' modelBasedQCPlots(data=testResultOneComparison, type="QQPlots", address="")
 #' # residual plots
 #' modelBasedQCPlots(data=testResultOneComparison, type="ResidualPlots", address="")
+#' PLEASE USE groupComparisonQCPlots function for this functionality. 
+#' modelBasedQCPlots WILL BE DEPRECATED.
 #' 
+
 modelBasedQCPlots = function(
     data, type, axis.size = 10, dot.size = 3, width = 10, height = 10, 
-    which.Protein = "all", address = ""
+    which.Protein = "all", address = "", displayDeprecationMessage = TRUE
 ) {
+
+    if(displayDeprecationMessage){
```

Comment:
Could we put this deprecation message at the end of the function?  Users may miss it if it's at the beginning as log messages printed later on can obscure this message